src/components/PatronForm/PatronForm.test.js

```diff
@@ -0,0 +1,192 @@
+import {
+  fireEvent,
+  render,
+  screen,
+} from '@testing-library/react';
+
+import '../../../test/jest/__mock__';
+
+import PatronForm, {
+  getIdentifier,
+} from './PatronForm';
+
+jest.mock('react-final-form', () => ({
+  ...jest.requireActual('react-final-form'),
+  // eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions
+  Field: jest.fn(({ onChange, ...rest }) => <div onClick={onChange} {...rest} />),
```

Comment:
you can use `input` instead of `div`. In this case, you do not need to add disabling of ESLint rule.